_posts/2022-06-17-ALP-Community-update.md

```diff
@@ -0,0 +1,36 @@
+---
+
+author: Lubos Kocman
+date: 2022-06-17 12:00:00+00:00
+layout: post
+license: CC-BY-SA-3.0
+title: Community work group update post oSCON22
+image: /wp-content/uploads/2022/06/opensuse-alp-concept-unofficial.png
+categories:
+- Announcements
+- openSUSE
+- Community
+tags:
+- openSUSE
+- Open Source
+- Community
+- Work Group
+- Open
+- Linux
+- AdaptableLinuxPlatform
+
+---
+
+[Community workgroup](https://en.opensuse.org/openSUSE:ALP/Workgroups/Community) for ALP would like to update you on what has happened since the openSUSE Conference 2022. Make sure to check the panel discussion with the ALP steering committee and other [talks](https://www.youtube.com/playlist?list=PL_AMhvchzBae22JG6CeBsSpjdrgMaeyN7) from oSCON22 as they become public.
```

Comment:
The [Community workgroup for ALP](https://en.opensuse.org/openSUSE:ALP/Workgroups/Community) would like to update you on what has happened since the openSUSE Conference 2022. Make sure to check the panel discussion with the ALP steering committee and other [talks](https://www.youtube.com/playlist?list=PL_AMhvchzBae22JG6CeBsSpjdrgMaeyN7) from oSCON22 as they become public.